### DISCUSSION TITLE
Simultaneous access to different accounts in different browser tabs not working?

### DISCUSSION THREAD

**Discussion by ozgurkazancci** [the-djmaze/snappymail#1288] [Q&A]
Hello there.

_"Allows for adding multiple accounts to primary one, simultaneous access to different accounts in different browser tabs is supported. Additional identities."_ was the feature I was completely interested in.

Installed SnappyMail in my server and added two domains/email accounts.

However, accessing different accounts in different tabs doesn't work at all. For instance;

The first tab, got myemail@domain1.com logged in, and when I open the second tab, and click to the upper-right icon to change the account to, let's say; myemail@domain2.com, the first tab automatically gets refreshed and changed to the 2nd account as well.

So both tabs show the same account. Any clue would be much appreciated.

Many thanks!

**Comment by CyberMinion** [comment]
Same issue here. In Firefox, whatever mailbox I last selected, is active. In the second tab, it will still show the previous mailbox, but attempting to open an email causes that tab to reload, bringing up the last-selected mailbox.

Also, it seems that if I navigate to the page "2" or beyond of emails, in the tab still viewing the "old" mailbox, it breaks my entire session, causing both tabs to log out.

**Comment by cm-schl** [comment]
Hi,

unfortunately I can only give you a reference to #837 and https://github.com/the-djmaze/snappymail/issues/650#issuecomment-1309856493 - this is not possible. The only workarounds are using different browsers or an incognito tab after logging in to SnappyMail multiple times.

> "Allows for adding multiple accounts to primary one, simultaneous access to different accounts in different browser tabs is supported. Additional identities." was the feature I was completely interested in.

Could you tell me where you've read this? This should be removed to not give new users false expectations...
